### PR TITLE
initial change - add pipeline entry to kubeflow registry

### PR DESCRIPTION
**Pull request by IronPan**
Mostly copied from existing registry
https://github.com/kubeflow/pipelines/tree/master/ml-pipeline/ml-pipeline

- Removed argo from the pipeline package. Rely on kubeflow argo instead
- Keep Spartakus entry for tracking pipeline specific usage

I'll have a separate PR to integrate the one-click and kfctl deployment

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1973)

### CODE REVIEW

**Comment by IronPan** [comment]
The tests failed with formatting error. Is there any tool I can use to format jsonnet code?

**Comment by kunmingg** [comment]
@IronPan 
Can you try this one?
https://github.com/kubeflow/kubeflow/blob/master/scripts/autoformat_jsonnet.sh

File: kubeflow/pipeline/all.libsonnet [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  parts(_env, _params):: {
+    local params = _env + _params,
+
+    local minio = import "kubeflow/pipeline/minio.libsonnet",
+    local mysql = import "kubeflow/pipeline/mysql.libsonnet",
+    local pipeline_apiserver = import "kubeflow/pipeline/pipeline-apiserver.libsonnet",
+    local pipeline_scheduledworkflow = import "kubeflow/pipeline/pipeline-scheduledworkflow.libsonnet",
+    local pipeline_persistenceagent = import "kubeflow/pipeline/pipeline-persistenceagent.libsonnet",
+    local pipeline_ui = import "kubeflow/pipeline/pipeline-ui.libsonnet",
+    local spartakus = import "kubeflow/pipeline/spartakus.libsonnet",
```
Comment (kunmingg) [inline kubeflow/pipeline/all.libsonnet]: spartakus is also available here: https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prototypes/spartakus.jsonnet
Can we merge this two?

Comment (IronPan) [reply kubeflow/pipeline/all.libsonnet]: Will pipeline always be installed? If so maybe we can remove it .

Comment (jlewi) [reply kubeflow/pipeline/all.libsonnet]: The plan is to install pipelines by default.
@abhi-g is looking into how we can provide additional information (e.g. about installed components) with spartakus.

So maybe we can remove spartakus for now and look into how to add pipeline specific data to that.

Comment (jlewi) [reply kubeflow/pipeline/all.libsonnet]: The plan is to install pipelines by default.
@abhi-g is looking into how we can provide additional information (e.g. about installed components) with spartakus.

So maybe we can remove spartakus for now and look into how to add pipeline specific data to that.

Comment (IronPan) [reply kubeflow/pipeline/all.libsonnet]: SG. Removed it.

**Pull request merged**